.env.example

```diff
@@ -12,3 +12,4 @@ DB_PATH=:memory: #required
 DEPLOY_START=0
 DEPLOY_END=19
 # FORK=mainnet
+TEST_ONLY=false
```

Comment:
what's this for?

Reply:
https://github.com/para-space/paraspace-core/pull/207#discussion_r1022288561
@GopherJ commented here